packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -199,10 +200,6 @@ export const getMemberExpressionMeta = (
   let bindingIdentifier: t.Identifier | null = null;
   let originalBindingType: t.Expression['type'] = 'Identifier';
 
-  if (t.isIdentifier(expression.property)) {
```

Comment:
Removed this as it duplicated the last item in the access path. Making it hard to tell if the member path had been fully traversed. We already do this on line: 219.